### ISSUE TITLE
in iOS on InAppWebView alwaysBounceVertical: true, disallowOverScroll: false do not work.  It never bounces.

### ISSUE THREAD

**Issue by delay** [pichillilorenzo/flutter_inappwebview#517]
- [x] I have read the [README](https://github.com/pichillilorenzo/flutter_inappwebview/blob/master/README.md)
- [x] I have already searched for the same problem
- [x] I have read and done the setup for Android: [IMPORTANT Note for Android and iOS](https://github.com/pichillilorenzo/flutter_inappwebview#important-note-for-android-and-ios) and [IMPORTANT Note for Android](https://github.com/pichillilorenzo/flutter_inappwebview#important-note-for-android)
- [x] I have read and done the setup for iOS: [IMPORTANT Note for Android and iOS](https://github.com/pichillilorenzo/flutter_inappwebview#important-note-for-android-and-ios) and [IMPORTANT Note for iOS](https://github.com/pichillilorenzo/flutter_inappwebview#important-note-for-ios)
- [x] I have ran the sample app and it does not work there

## Environment

| Technology           | Version       |
| -------------------- | ------------- |
| Flutter version      |    Channel dev, 1.22.0-9.0.pre           |
| Plugin version       |    ^4.0.0+4           |
| Android version      |               |
| iOS version          |     13.7          |
| Xcode version        |   11.7            |

Device information: iPhone X

## Description
There is no bounce on InAppWebView on iOS like there is in native safari.   I have enabled these two settings. alwaysBounceVertical: true, disallowOverScroll: false

**Expected behavior:** 
I expect it to bounce when reaching the bottom or top of screen when scrolling.
**Current behavior:** 
It doesn't bounce at all.
## Steps to reproduce
Project to reproduce issue.
```dart
import 'dart:async';
import 'package:flutter/material.dart';
import 'package:flutter_inappwebview/flutter_inappwebview.dart';

Future main() async {
  WidgetsFlutterBinding.ensureInitialized();
  runApp(new Browser());
}

class Browser extends StatefulWidget {
  @override
  _BrowserState createState() => new _BrowserState();
}

class _BrowserState extends State<Browser> {
  InAppWebViewController _webViewController;
  String url = "";
  double progress = 0;

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: Scaffold(
        appBar: AppBar(
          title:
              Text("${(url.length > 50) ? url.substring(0, 50) + "..." : url}"),
        ),
        body: InAppWebView(
          initialUrl: "https://google.com",
          initialOptions: InAppWebViewGroupOptions(
              ios: IOSInAppWebViewOptions(
                  alwaysBounceVertical: true, disallowOverScroll: false),
              crossPlatform: InAppWebViewOptions(
                debuggingEnabled: true,
              )),
          onWebViewCreated: (InAppWebViewController controller) {
            _webViewController = controller;
          },
          onLoadStart: (InAppWebViewController controller, String url) {
            setState(() {
              this.url = url;
            });
          },
          onLoadStop: (InAppWebViewController controller, String url) async {
            setState(() {
              this.url = url;
            });
          },
          onProgressChanged: (InAppWebViewController controller, int progress) {
            setState(() {
              this.progress = progress / 100;
            });
          },
        ),
      ),
    );
  }
}
```

1. This
2. Than that
3. Then

**Comment by ruizyi** [comment]
Is it fixed it or ?

**Comment by chavesgu** [comment]
same issues

**Comment by delay** [comment]
This was just fixed in the latest build! fixed with the latest release 5.0.5+1

_Originally posted by @pichillilorenzo in https://github.com/pichillilorenzo/flutter_inappwebview/issues/696#issuecomment-788216651_
